Comparison shopping engine/Scrapers/Selenium/AbstractSeleniumScraper.cs

```diff
@@ -84,7 +84,7 @@ public void ScrapeWithSelenium(object s)
                                                 products.Add(new Product(name.Replace("'", "''"), price, productUrl,
                                                     photoUrl,
                                                     "None",
-                                                    site + ".lt"));
+                                                    site)); // + ".lt"
```

Comment:
Kokiu tikslu nuimtas nuo produkto šaltinio puslapio .lt galas?

Reply:
Darant bigbox selenium scraperi, atsirado problema, jog pridedinėjo itemus kaip bigbox.lt, o ne bigbox.
Kadangi visi kiti saltiniai buvo be .lt galo tai nuėmiau si gala, is viskas veikia, nes nedėjo pirma i database bigbox produktu. Finalas toks jog palikau uzkomentaves, nes gali atsirasti koks bugas su kitais scraperiais, bet kiek testavau problemų nebuvo. Kiek pamenu sitas .lt galas buvo reikalingas, kuomet dar senoje mainformoje dėjome produktus i listview ir tenais parasydavom is kokio tinklapio paimta su .lt, bet dabar to nebedarom.

Reply:
Nežinau kaip tai galėjo turėti įtakos dėjimui i db
![image](https://user-images.githubusercontent.com/67711718/97223751-de9ae500-17d8-11eb-984e-fddfe6c485eb.png)

Kaip nuotraukoje matosi, dedant i db nurodomas "site" kas yra tik tinklapio pavadinimas, be jokių .lt ar www., mano komentuotam kode aš jeigu gerai pamenu kaip veikia, tai ten tiesiog dėl estetikos, kad pridejus i listview'a rasytų *.lt